geonode_mapstore_client/client/static/mapstore/configs/localConfig.json

```diff
@@ -350,6 +350,7 @@
                             "value": "layer",
                             "type": "link",
                             "href": "/createlayer/",
+                            "config": "createLayer",
```

Comment:
@DavidQuartz could we try to use disableIf instead to add a new param?

`"disableIf": "{((state('settings') && state('settings').createLayer) ? true : false)}",`

I did not test this so please try it locally if it works